frontend/app/src/App.tsx

```diff
@@ -1554,6 +1554,9 @@ export class App extends PureComponent<Props, State> {
           pageName,
           fragmentId,
           isAutoRerun,
+          contextInfo: {
+            timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
```

Comment:
something to consider from product side: in addition to `timezone`, we could also provide the `timezoneOffset` (via `new Date().getTimezoneOffset();`). Sometimes one or the other is easier to work with.

Reply:
Yeah, absolutely

Reply:
Added, just FYI, the `new Date().getTimezoneOffset()` could be unintuitive a little bit, since it
> returns the difference, in minutes, between this date as evaluated in the UTC time zone, and the same date as evaluated in the local time zone

So for example for Yerevan (`GMT +4`, it is `-240`), but I think if we highlight it in the docs, this property could be helpful for Stremalit app developers. 

CC: @sfc-gh-dmatthews